pkg/sfu/downtrack.go

```diff
@@ -30,6 +32,15 @@ const (
 	RTPBlankFramesMax             = 6
 )
 
+type SequenceNumberOrdering int
+
+const (
+	SequenceNumberOrdering_CONTIGUOUS = iota
```

Comment:
set const type, and camelcase style
SequenceNumberOrderingContiguous SequenceNumberOrdering = iota

Reply:
@cnderrauber Is that the preferred style in golang? Then I should change my signal/state definitions in `StreamAllocator` too.

Reply:
Yes, it is golang naming convention

Reply:
Got it. Will change here. Will change the `StreamAllocator` stuff later.